index.js

```diff
@@ -5,7 +5,8 @@ const defaultOptions = {
   moduleName: 'oauth',
   fetchUser: () => ({}),
   onLogout: () => {},
-  scopes: []
+  scopes: [],
+  component: null
```

Comment:
Could you name this something more specific, like `loadingPagePath` or something that a) specifies what the component is used for and b) specifies that it's a path, not an object or whatever